integrations-and-supported-tools/catalyst/notebooks/Neptune_Catalyst.ipynb

```diff
@@ -358,19 +378,18 @@
    "outputs": [],
    "source": [
     "my_runner.log_artifact(\n",
-    "    path_to_artifact=\"./checkpoints/best.pth\", tag=\"best_model\", scope=\"experiment\"\n",
+    "    path_to_artifact=\"./checkpoints/model.best.pth\", tag=\"best_model\", scope=\"experiment\"\n",
```

Comment:
This doesn't look alright. 
Perhaps, it should be `model_best.pth` instead of `model.best.pth`

Reply:
nope... its indeed `model.best.pth`
![image](https://user-images.githubusercontent.com/41324509/211304438-9139b59e-5950-4991-814c-7f5abb2e6d50.png)